rex-release/out.sh

```diff
@@ -62,7 +62,7 @@ upload-release() {
     from_exists=$(release-file-exists "$version" "$from")
     to_exists=$(release-file-exists "$version" "$to")
 
-    if [ -n "$from_exists" ] || [ -z "$to_exists" ]; then
+    if [ -n "$from_exists" ] || { [[ "$to" != /books* ]] && [ -z "$to_exists" ]; }; then
```

Comment:
I've never seen the curlies used for grouping before, do they do something different from parents?

https://stackoverflow.com/a/14965071

Reply:
the AI teaching me bash lied